content/SCALE/SCALEUIReference/Apps/InstallCustomAppScreens.md

```diff
@@ -0,0 +1,274 @@
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppPortForwarding.png" alt="Port Forwarding Settings" id="10: Port Forwarding Settings" >}}
+
+{{< expand "Settings Information" "v" >}}
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Specify Node ports to forward to workload** | Use to specify one or more local ports to forward to a container (pod). Click **Add** to display a block of **Port Forwarding Configuration** settings. |
+| **Container Port** | Required. Do not enter the same port number used by another system service or container. See [Default Ports]({{< relref "DefaultPorts.md" >}}) for a list of assigned ports in TrueNAS. |
+| **Node Port** | Required. Enter a node port number over **9000**. |
+| **Protocol** | Select the protocol from the dropdown list. Options are **TCP Protocol** or **UDP Protocol**.  |
+{{< /truetable >}}
+{{< /expand >}}
+
+### Storage Settings
+The **Storage** settings specify persistent host paths and share data that separate from the lifecycle of the container. 
+Create the storage volumes in SCALE and set the host path volume to a dataset and directory path. 
+You can mount SCALE storage locations inside the container with host path volumes. Define the path to the system storage and the container internal path for the system storage location to appear.
+For more details, see the [Kubernetes hostPath documentation](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
+Users can create additional Persistent Volumes (PVs) for storage within the container. 
+PVs consume space from the pool chosen for application management. To do this, name each new dataset and define a path where that dataset appears inside the container.
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppScreenStorage.png" alt="Storage Settings" id="11: Storage Settings" >}}
+
+{{< expand "Settings Information" "v" >}}
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppAddHostPathVol.png" alt="Host Path Volume Settings" id="12: Host Path Volume Settings" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Host Path Volumes** | Use to configure a persistent host path volume. Click **Add** to display a block of **Host Path Volume** settings. |
+| **Host Path** | Required. Enter or click <span class="material-icons">arrow_right</span> to the left of <span class="material-icons">folder</span> **/mnt** to browse to the location of the dataset to populate the **Mount Path**. Click on the dataset to select and display it in the **Mount Path** field. |
+| **Mount Path** | Required. Enter the <file>**/data**</file> directory where the host path mounts inside the pod. |
+| **Read Only** | Select to make the mount path inside the pod read-only and prevent using the container to store data. Kubernetes recommends scoping it to only require file or directory and mounting it as read only. |
```

Comment:
Is "scoping" correct here or a typo?

Reply:
That is what Kubernetes docs says so scoping is correct